ios/brave-ios/Sources/Brave/Frontend/Browser/Helpers/TabBrowserData.swift

```diff
@@ -266,6 +266,7 @@ class TabBrowserData: NSObject, TabObserver {
 
   var translateHelper: BraveTranslateTabHelper?
   private(set) lazy var leoTabHelper = BraveLeoScriptTabHelper(tab: tab)
+  private(set) lazy var youtubeQualityTabHelper = YoutubeQualityTabHelper(tab: tab)
```

Comment:
Still not the correct way to do this. (Leo/translateHelper aren't correct either, but likely there because they were stored on Tab at the time of refactor).  Check helpers that extend `TabDataValues` like `PlaylistTabHelper`